bokehjs/src/coffee/source/geojson_data_source.coffee

```diff
@@ -0,0 +1,124 @@
+_ = require "underscore"
+{logger} = require "../common/logging"
+ColumnDataSource = require "./column_data_source"
+
+class GeoJSONDataSource extends ColumnDataSource.Model
+
+  initialize: (options) ->
+    @set('geojson', JSON.parse(options.geojson))
```

Comment:
Thanks @havocp, that was very helpful, and I've pushed a commit inline with your suggestions - which makes total sense. But you've highlighted that this isn't exactly what I wanted to do anyway - please see comment on main thread.